CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## [unreleased]
 
+ * K8S Remote App Scan should notice when service port changes and load app definition from the new URL.
```

Comment:
A ChangeLog entry should have the format [Plugin]: What have changed - so in this case it would be:

Kubernetes Remote App Registry: If the service port changes the App definition gets reloaded with the next scan